apps/src/code-studio/teacher.js

```diff
@@ -0,0 +1,39 @@
+/* globals dashboard  */
+/**
+ * JS used by _teacher.html.haml
+ */
+import $ from 'jquery';
+
+function resizeScrollable() {
+  var newHeight = $('.teacher-panel').innerHeight() -
+      $('.teacher-panel h3').outerHeight() -
+      15 - // magic..
+      $('.non-scrollable-wrapper').outerHeight();
+  $('.scrollable-wrapper').css('max-height', newHeight);
+}
+
+$(document).ready(() => {
+  $(window).resize(dashboard.utils.debounce(resizeScrollable, 250));
```

Comment:
I'm pretty sure you can import individual lodash functions, so you don't have to bring in all of lodash.